tests/tui/test_seed_based.py

```diff
@@ -0,0 +1,115 @@
+# -*- coding: utf-8 -*-
+
+import os
+import shutil
+from functools import partial
+from pathlib import Path
+
+from .pilot_functions import (
+    _load_data,
+    _set_work_dir,
+    add_atlas_or_seed_or_map_file_pattern,
+    add_new_feature,
+    check_and_run_tab_refresh,
+    select_images,
+    settable_scroll_screen_down,
+)
+
+
+async def run_before(pilot, data_path=None, work_dir_path=None, stage=None, file_pattern=None) -> None:
+    # always reload the app first, there is some strange crossinteraction between tests, nothing else helped except using
+    # -n 2 flag for the pytest, i.e., running each test with a separate worker
+    how_much_down = 35
+
+    pilot.app.reload_ui()
+    if isinstance(data_path, Path):
+        data_path = str(data_path)
+    if isinstance(work_dir_path, Path):
+        work_dir_path = str(work_dir_path)
+    if isinstance(file_pattern, Path):
+        file_pattern = str(file_pattern)
+    # Delete work_dir if exists
+    if os.path.exists(work_dir_path):
+        shutil.rmtree(work_dir_path)
+    print("----------------------------", data_path, work_dir_path)
+
+    # Define functions to execute based on stage requirements
+    async def add_seed_based_task():
+        await add_new_feature(pilot, feature_type="seed_based", label="seed_based_1")
+        # select all images
+        await select_images(pilot)
+        # click on "Add" (seed images)
+        await add_atlas_or_seed_or_map_file_pattern(pilot, file_pattern)
+        #
+        # deselect second seed file
+        # await pilot.click(offset=(71, 33))
+        await pilot.click("#tag_selection")
+        await pilot.press("down")
+        await pilot.press("down")
+        await pilot.press("enter")
+        #
+        # # change minimum coverage from 0.8 to 0.85
+        # await pilot.click(offset=(131, 40))
+        await pilot.click(pilot.app.get_widget_by_id("minimum_coverage").get_widget_by_id("input_label_input_box"))
+        await pilot.press("5")
+        # press tab to unfocus the input box
+        await pilot.press("tab")
+
+    async def final_stage_tasks():
+        # click somewhere outside of the form area
+        # await pilot.click(offset=(50, 10))
+        await check_and_run_tab_refresh(pilot)
+        await settable_scroll_screen_down(pilot, how_much_down)
```

Comment:
Is it possible to click a button by ID without scrolling, so for example a button that is currently not visible?

Reply:
no, that you can target only that the user could see